packages/@okta/vuepress-site/docs/guides/device-authorization-grant/main/index.md

```diff
@@ -0,0 +1,154 @@
+1. On your native application page, fill in the application settings. Ensure that you select **Device Authorization** as the allowed **Grant type** in the General Settings.
+
+> **Note:** Device Authorization is only supported for use with a native application.
+
+## Configure the Authorization Server policy rule for Device Authorization
+
+Both Org and Custom Authorization Servers support Device Authorization. Ensure that Device Authorization is enabled at the policy rule level.
+
+To check that Device Authorization is enabled:
+
+1. In the left navigation pane of the Admin Console, go to **Security** > **API** and select the "default" Custom Authorization Server. Note that the examples in this guide use the "default" Custom Authorization Server.
+1. On the **Access Policies** tab, select the access policy that you want to configure Device Authorization for.
+1. Click the pencil icon for the Default Policy Rule.
+1. In the Edit Rule dialog box, select **Device Authorization** for the grant type and click **Update Rule**.
+
+## Configure the smart device
+
+### Request the device verification code
+
+The smart device first needs to call the `/device/authorize` endpoint to obtain the unique verification code.
+
+**Request example**
+
+```bash
+  curl --request POST \
+  --url https://${yourOktaDomain}/oauth2/default/v1/device/authorize \
+  --header 'Content-Type: application/x-www-form-urlencoded' \
+  --data-urlencode 'client_id={clientId}' \
+  --data-urlencode 'scope=openid profile offline_access'
+```
+
+The POST request passes the following parameters:
+
+* `client_id`, which matches the Client ID of the OAuth 2.0 application that you created
+* `scope` to specify which access privileges are being requested for access tokens. See [Scopes](https://developer.okta.com/docs/reference/api/oidc/#scopes) for a list of supported scopes.
+
+**Response example**
+
+```json
+{
+   "device_code": "f5ec0430-c2fe-4a93-912d-08631f1effed",
+   "user_code": "HMBSRJRC",
+   "verification_uri": "https://${yourOktaDomain}/activate",
+   "expires_in": 600,
+   "interval": 5
+}
+```
+
+The properties in the response are:
+
+* `device_code`: The long string that the device uses to exchange for an access token.
+* `user_code`: The text that you enter at the URL that is listed as the value for `verification_uri`.
+* `verification_uri`: The URL that the user needs to enter into their phone to start signing in.
+* `expires_in`: The number of seconds that this set of values is valid. After the device code and user code expire, the user has to start the device verification process over.
+* `interval`: The number of seconds that the device should wait between polling to see if the user has finished signing in.
+
+The `user_code` and `verification_uri` must appear on the smart device for the user.
+
+> **Note:** A QR code compliant `verification_uri` is coming soon.
```

Comment:
I'll suggest a slight re-phrasing;
```suggestion
> **Note:** A QR code compliant verification uri is coming soon.
```
Reason is that the new QR compliant verification uri will be called `verification_uri_complete` and we'll still keep the existing `verification_uri` the way it is described in this doc, so it'll be confusing to mention that property specifically in this note.